package.json

```diff
@@ -89,20 +89,11 @@
     "in-publish": "^2.0.0",
     "jsdoc": "^3.4.0",
     "karma": "^1.7.1",
-    "karma-browserstack-launcher": "^1.3.0",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-coverage": "^1.1.1",
-    "karma-detect-browsers": "^2.2.6",
-    "karma-firefox-launcher": "^1.1.0",
-    "karma-qunit": "^1.2.1",
-    "karma-safari-launcher": "^1.0.0",
-    "karma-safaritechpreview-launcher": "0.0.6",
     "karma-spec-reporter": "0.0.32",
     "lodash": "^4.17.4",
-    "lodash-compat": "^3.10.0",
     "npm-run-all": "^4.1.2",
     "portscanner": "^2.1.1",
-    "qunitjs": "^2.0.1",
+    "qunit": "2.4.1",
```

Comment:
`videojs-generate-karma-config` comes with its own `qunit` https://github.com/videojs/videojs-generate-karma-config/blob/master/package.json#L49. Instead of serving the `test/index.html` the other projects start up a karma server, see https://github.com/videojs/mpd-parser/pull/28/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R21 . cc @BrandonOCasey

Reply:
I noticed that video.js [still has qunit](https://github.com/videojs/video.js/blob/master/package.json#L119) so I left it in. 

Also versions above 2.4.1 broke the tests in some non-obvious way so I wanted to lock the version down to 2.4.1. See:  https://travis-ci.org/videojs/http-streaming/builds/449537117. I tried ensuring that muted was on the video element, and running Chrome with the autoplay-policy set to `no-user-gesture-required` but neither made a difference. 

That said, maybe using qunit's test page rather than `test/index.html` will fix things. I'll give it a shot

Reply:
Yeah, not sure why we still have qunit in Video.js directly since the karma-config project includes it.

It looks like the tests that failed in that travis run may be autoplaying players that weren't muted? That's what it sounds like from the message at least.

Reply:
I think the messages were red herrings since disabling the autoplay policy via flags made no difference

Reply:
Looks like qunit is required at the top level because of karma. Without it:

>>>> karma start scripts/karma.conf.js

module.js:550
    throw err;
    ^

Error: Cannot find module 'qunit'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.resolve (internal/module.js:18:19)
    at initQUnit (/Users/ldayananda/Documents/work/vhs/node_modules/karma-qunit/lib/index.js:14:39)
    at Array.invoke (/Users/ldayananda/Documents/work/vhs/node_modules/di/lib/injector.js:75:15)
    at Injector.get (/Users/ldayananda/Documents/work/vhs/node_modules/di/lib/injector.js:48:43)
    at /Users/ldayananda/Documents/work/vhs/node_modules/karma/lib/server.js:143:20
    at Array.forEach (<anonymous>)
    at Server._start (/Users/ldayananda/Documents/work/vhs/node_modules/karma/lib/server.js:142:21)
    at Injector.invoke (/Users/ldayananda/Documents/work/vhs/node_modules/di/lib/injector.js:75:15)
    at Server.start (/Users/ldayananda/Documents/work/vhs/node_modules/karma/lib/server.js:103:18)
    at Object.exports.run (/Users/ldayananda/Documents/work/vhs/node_modules/karma/lib/cli.js:280:26)
    at Object.<anonymous> (/Users/ldayananda/Documents/work/vhs/node_modules/karma/bin/karma:3:23)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3

Reply:
could be pinned to ~2.5.1 instead